client/src/api/queries/event.ts

```diff
@@ -57,3 +62,12 @@ export const createEventAPI = (organizationId: number, data: CreateEventAPIReque
 const getEventDetailAPI = (eventId: number) => {
   return fetcher.get<EventDetail>(`organizations/events/${eventId}`);
 };
+
+export const postEventParticipation = (
+  eventId: number,
+  answers: { questionId: number; answerText: string }[]
```

Comment:
p2.
요렇게 사용해도 좋을 것 같네용
```suggestion
  answers: Answer[]
```